### PR TITLE
Add initial link support

### PR DESCRIPTION
**Pull request by vejmarie**
Thank you for creating a pull request to contribute to FreeCAD! To ease integration, please confirm the following:

- [ ] Branch rebased on latest master `git pull --rebase upstream master`
- [ ] Unit tests confirmed to pass by running `./bin/FreeCAD --run-test 0`
- [ ] Commit message is [well-written](https://chris.beams.io/posts/git-commit/)
- [ ] Commit message includes `issue #<id>` or `fixes #<id>` where `<id>` is the [associated MantisBT](https://freecadweb.org/wiki/tracker#GitHub_and_MantisBT) issue id if one exists

And please remember to update the Wiki with the features added or changed once this PR is merged.  
**Note**: If you don't have wiki access, then please mention your contribution on the [0.19 Changelog Forum Thread](https://forum.freecadweb.org/viewtopic.php?f=10&t=34586).

---

### CODE REVIEW

File: src/Mod/Arch/importOBJ.py [hunk 1]
```diff
@@ -72,14 +72,23 @@ def getIndices(obj,shape,offsetv,offsetvn):
...
             except: # unimplemented curve type
-                myshape = obj.Shape.copy(False)
-                myshape.Placement=obj.getGlobalPlacement()
+                if obj.isDerivedFrom("App::Link"):
+                  if obj.Shape:
+                   myshape = obj.Shape.copy(False)
+                   myshape.Placement=obj.LinkPlacement
+                  else:
+                   myshape = obj.Shape.copy(False)
+                   myshape.Placement=obj.getGlobalPlacement()
```
Comment (vocx-fc) [inline src/Mod/Arch/importOBJ.py:91]: Please add proper 4 space indentation in the code that you are modifying. Also, please be aware that now the Draft and Arch workbenches have coding standards. Basically, "follow PEP8".

File: src/Mod/Arch/importOBJ.py [hunk 2]
```diff
@@ -157,7 +166,7 @@ def export(exportList,filename,colors=None):
     materials = []
     outfile.write("mtllib " + os.path.basename(filenamemtl) + "\n")
     for obj in objectslist:
-        if obj.isDerivedFrom("Part::Feature") or obj.isDerivedFrom("Mesh::Feature"):
+        if obj.isDerivedFrom("Part::Feature") or obj.isDerivedFrom("Mesh::Feature") or obj.isDerivedFrom("App::Link") or obj.isDerivedFrom("App::Link"):
```
Comment (vocx-fc) [inline src/Mod/Arch/importOBJ.py:169]: Twice derived from "App::Link".

Comment (vejmarie) [reply src/Mod/Arch/importOBJ.py]: Sorry, just fixed it

File: src/Mod/Arch/importOBJ.py (hunk 1 above)
Comment (vejmarie) [reply src/Mod/Arch/importOBJ.py]: Sorry, I am definitely not a python developper (C++ or Golang) and do my best.

Comment (vocx-fc) [reply src/Mod/Arch/importOBJ.py]: Awesome. We need more C++ programmers because that's where the real meat is, Part and PartDesign.

**Pull request merged**